scripts/import_currencies.py

```
@@ -0,0 +1,75 @@
+}
+
+BASE_ID = os.environ["NOCODB_BASE_ID"]
+TABLE_ID = os.environ["NOCODB_TABLE_ID"]
+
+h = {"xc-token": API_KEY, "Content-Type": "application/json"}
+
+with open("currencies.json") as f:
+    currencies = json.load(f)
+
+# Fetch all existing codes in one request
+existing_resp = requests.get(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)